src/IECoreMaya/ToMayaMeshConverter.cpp

```diff
@@ -344,6 +346,123 @@ bool ToMayaMeshConverter::doConversion( IECore::ConstObjectPtr from, MObject &to
 		}
 	}
 
+	/// Add corners and edge creases
+
+	MUintArray cornerIdsMaya;
+	MDoubleArray cornerSharpnessMaya;
+
+	const IECore::IntVectorData *cornerIdsData = mesh->cornerIds();
+	const std::vector<int> cornerIds = cornerIdsData->readable();
+
+	if( !cornerIds.empty() )
+	{
+		const IECore::FloatVectorData *cornerSharpnessesData = mesh->cornerSharpnesses();
+		const std::vector<float> cornerSharpnesses = cornerSharpnessesData->readable();
+
+		for( size_t i = 0; i < cornerIds.size(); ++i )
+		{
+			cornerIdsMaya.append( cornerIds[i] );
+			cornerSharpnessMaya.append( cornerSharpnesses[i] );
```

Comment:
I stumbled when reading the word `sharpnesses` in the first commit and I noticed @mattigruener just uses `sharpness` here as does Arnold (`crease_sharpness`) while Alembic and USD use `sharpnesses`. @johnhaddon did you give any consideration to ditching the `es` for Cortex?

Reply:
I did. I felt the plural was more appropriate, because we're storing multiple sharpnesses.